common/is/clean-database.sql

```diff
@@ -6,5 +6,8 @@ TRUNCATE TABLE IDN_AUTH_TEMP_SESSION_STORE;
 TRUNCATE TABLE IDN_OAUTH2_AUTHORIZATION_CODE;
 TRUNCATE TABLE IDN_OAUTH2_ACCESS_TOKEN_SCOPE;
 TRUNCATE TABLE IDN_OAUTH2_ACCESS_TOKEN;
+TRUNCATE TABLE IDN_AUTH_SESSION_META_DATA;
+TRUNCATE TABLE IDN_AUTH_SESSION_APP_INFO;
+TRUNCATE TABLE IDN_AUTH_USER_SESSION_MAPPING;
```

Comment:
Shall we add these to the MSSQL script as well? As I saw these are not present there.

Reply:
As far as I know @ashendes is working on that. @ashendes please confirm.

Reply:
Will add these.

Reply:
Added with 6494e38e9f2e174834780416a23942b407a19739

Reply:
Added with https://github.com/wso2/performance-is/pull/107